tools/import/import.css

```diff
@@ -108,10 +108,21 @@
  padding-top: var(--spacing-s);
 }
 
-.import div.hidden {
+.import .hidden {
   display: none;
 }
 
+.import button.negative,
+.import button.negative:focus {
+  background-color: var(--red-800);
+  color: var(--gray-100);
+}
+
+.import button.negative:hover {
+  color: var(--color-text);
+  border-color: var(--red-800);
+}
```

Comment:
If you want to go down the path of intriducing a negative variant I think there are more states you need to handle. Also, an outline button may be more appropriate.

https://spectrum.adobe.com/page/button/#Negative-variant

<img width="150" alt="image" src="https://github.com/user-attachments/assets/cdc6ef5a-225c-483a-bff3-7155ba6b6b01">

<img width="143" alt="image" src="https://github.com/user-attachments/assets/d27fdd27-1254-473f-bee4-0f069973f1f4">

Reply:
```
.import button.negative, .import button.negative:focus {
    background-color: var(--grey-100);
    color: var(--red-800);
    border-color: var(--red-800);
}

.import button.negative:focus {
    outline-color: var(--red-800);
}

.import button.negative:hover {
    background-color: var(--red-100);
    color: var(--red-800);
}

```

Reply:
<img width="136" alt="image" src="https://github.com/user-attachments/assets/d2e2e792-17b7-42e4-96d6-c1baa0d867b0">